README.md

```diff
@@ -331,34 +332,34 @@ The tool for outputing fixed flexbox bugs codes.
 
 ## TypeScript
 Default settings are strict.
-Ask your frontend engineers if you want to relieve strict settings.
+Ask frontend engineers if you want to relieve strict settings.
 
 ## Vue.js
 Installed as default.
-Ask your frontend engineers if you want to use other JS frontend frameworks like jQuery, React, Angular etc.. .
-If you don't use Vue.js, remove that and that's ecosystem.
+Ask frontend engineers if you want to use other JS frontend frameworks like jQuery, React, Angular etc.. .
+If you don't use Vue.js, remove that.
 
 ## axios
 Use axios. Just axios. Don't use `$.ajax`.
 
 ## jest
-Testing frame work for JavaScript.
-This config is apart from webpack so ask your frontend engineers when you get stuck settings.
+Testing framework for JavaScript.
+This config is apart from webpack so ask frontend engineers when you get stuck in settings.
 
 # Q&A
-## Why not JQuery?
-I haven't installed JQuery due to DOM handling of pure JS enough good.
-Ask your frontend engineers when you think you have to install it.
+## Why not jQuery?
+I haven't installed jQuery due to DOM handling of pure JS enough good.
+Ask frontend engineers when you think you have to install it.
 
 ## Why not assets pipeline?
-assets pipeline outputs a file all JS and CSS.
-The code base of the project bigger, the output file bigger and it takes much time to download without cache.
-Additionaly, assets pipeline uses gem and it makes frontend settings ruined.
+Assets pipeline outputs all JS and CSS files.
+The code base of the project is bigger, the output file is bigger and it takes much time to download without cache.
```

Comment:
The code bigger code base of the project is, the bigger the output file is and ...